admin/linux/debian/debian.xenial/post-patches/qt5.5.patch

```diff
@@ -11,3 +11,14 @@ diff -ruN nextcloud-client-2.4.0.orig/src/CMakeLists.txt nextcloud-client-2.4.0/
  if (Qt5Core_VERSION VERSION_LESS 5.9.0)
  message(STATUS "For HTTP/2 support, compile with Qt 5.9 or higher.")
  endif()
+--- nextcloud-client-2.4.0.orig/admin/osx/CMakeLists.txt	2018-05-22 07:01:48.248646951 +0200
++++ nextcloud-client-2.4.0/admin/osx/CMakeLists.txt	2018-05-22 07:01:51.012689711 +0200
+@@ -11,7 +11,7 @@
+   set(MAC_INSTALLER_DO_CUSTOM_BACKGROUND "0")
+ endif()
+ 
+-find_package(Qt5 5.6 COMPONENTS Core REQUIRED)
++find_package(Qt5 5.5 COMPONENTS Core REQUIRED)
```

Comment:
Mmmm we do target 5.6 or above... so this could have unforeseen consequences...

Reply:
Yes, I agree. 

However, this patch is used for Xenial only, which has Qt 5.5 unfortunately (Xenial was released a short time before Qt 5.6 was out, so they decided to ship 5.5). I could provide a full set of Qt 5.6 packages in the PPA for Xenial, but that task seems daunting at best :) So requiring 5.5 seems to be a reasonable way to deal with the issue, as the client does seem to work with it. However, I am going to include note a in the PPA description that Xenial support is on a best effort basis.

In the longer run, if the client becomes really incompatible with Qt 5.5, either we need to provide the 5.6 packages in the PPA or Xenial support should be dropped (although Xenial is supported by Canonical until 2021...)

Reply:
Does it make sense to you @rullzer?

Reply:
yes fine by me